test/e2e/compatibility/cp_compatibility_kubernetes_multizone.go

```diff
@@ -44,32 +44,30 @@ func CpCompatibilityMultizoneKubernetes() {
 		globalCluster = NewK8sCluster(NewTestingT(), Kuma1, Silent).
 			WithTimeout(6 * time.Second).
 			WithRetries(60)
+		E2EDeferCleanup(func() {
+			Expect(globalCluster.DeleteKuma()).To(Succeed())
+			Expect(globalCluster.DismissCluster()).To(Succeed())
+		})
 
 		globalReleaseName = fmt.Sprintf(
 			"kuma-%s",
 			strings.ToLower(random.UniqueId()),
 		)
 
-		E2EDeferCleanup(func() {
```

Comment:
In this file nothing actually changed, right? No more or less fallible calls after `E2EDeferCleanup`?

Reply:
The only change is the order